### PR TITLE
Creates project 2.0

### PR DESCRIPTION
**Pull request by renatamottam**
In this PR I created two new pages for the project and updated the homepage.
|Home|About Us|More Ways to Help|
|-|-|-|
|image|screencapture-localhost-3000-about-us-2020-07-13-10_38_15|image|

### CODE REVIEW

**Review by hellofanny — APPROVED** [review]
Well done renatinha :)

**Comment by hellofanny** [comment]
I also added a temporary readme file. @danzanzini can you also check it please? Thank you 😊

File: app/views/ways_to_help/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+      <article class="ways_options">
+        <h3>Se puder, Fique em casa</h3>
+        <p>Seguir as diretrizes da saúde pública da sua região, apesar de obrigação, já é uma enorme ajuda. Entretanto, <strong>lembre-se que para a covid-19 você pode ser um vetor de transmissão</strong>, então, o máximo que você puder ficar em casa, melhor para desacelerar a transmissão da doença.</p>
+        <p>Fique em casa o máximo possível e, quando precisar sair, pratique as regras de distanciamento social e de higiene.</p>
+      </article>
+
+      <article class="ways_options">
+        <h3>Cuide de você</h3>
+        <p>Faça esforço para se manter saudável, para isso se alimente bem, descanse e evite atividades de alto risco. Assim, além de reduzir suas chances para contrair o coronavírus, você ajuda a evitar outras doenças e oprimir os sistemas de saúde.</p>
```
Comment (danzanzini) [inline app/views/ways_to_help/index.html.erb:144]: I would change this sentence to:

"Se mantenha saudável. Para isso, se alimente bem, beba muita água, descanse e evite atividades de alto risco. Assim, você evita outras doenças e acidentes que podem te levar a utilizar o sistema de saúde."

File: app/views/ways_to_help/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    <p><i>Muitas dessas ações podem ser simples para quem as toma, mas fazem muita diferença na vida das pessoas afetadas… inclusive, pode ser a sua própria vida!</i></p>
+  </section>
+
+  <section class="section section_ways">
+    <h2>Cuidados com a comunidade</h2>
+
+    <div class="content -larger / ways_comunity">
+      <article class="ways_options">
+        <h3>Explique</h3>
+        <p>Muitas pessoas não entendem a gravidade da doença ou a necessidade do isolamento social e sua posição como única “solução” que temos. Seja empático e apresente os argumentos de forma paciente e mostrando os dados.</p>
```
Comment (danzanzini) [inline app/views/ways_to_help/index.html.erb:16]: "Tenha empatia e apresente..."
"Tente entender o lado da pessoa e apresente..."

File: app/views/ways_to_help/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+          <p>Tenha cuidado com as informações que você repassa, cheque a confiabilidade das fontes e sempre leia/assista/escute antes de repassar. Muitas informações erradas podem ser prejudiciais em vários níveis: <strong>para a saúde</strong> (como informações ineficientes de prevenção ou medicação), <strong>para o psicológico</strong> (como um fluxo constante de informações pode afetar os níveis de ansiedade), entre outros.</p>
+        </article>
+      </article>
+
+      <article class="ways_options">
+        <h3>Divulgue iniciativas</h3>
+        <p>Além de informar sobre a doença (com dados verídicos!), precisamos deixar clara a realidade bastante agravada das pessoas já vulneráveis. E sua voz tem poder!</p>
+        <p>Compartilhe informações das instituições que precisam de ajuda, exponha a necessidade das pessoas.</p>
+        <p>Você pode divulgar entre seus familiares e amigos, use suas redes sociais. <strong>Mesmo que você não possa doar, sua divulgação pode inspirar algum conhecido à tomar essa ação.</strong></p>
```
Comment (danzanzini) [inline app/views/ways_to_help/index.html.erb:35]: "...divulgar entre seus familiares e círculos de amizade..."

Comment (danzanzini) [reply app/views/ways_to_help/index.html.erb:35]: "...inspirar alguém que você conheça à..."

File: app/views/ways_to_help/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        <h3>Cuide de você</h3>
+        <p>Faça esforço para se manter saudável, para isso se alimente bem, descanse e evite atividades de alto risco. Assim, além de reduzir suas chances para contrair o coronavírus, você ajuda a evitar outras doenças e oprimir os sistemas de saúde.</p>
+        <p><strong>Algumas dicas que fazem bem a sua saúde física e mental:</strong></p>
+        <ul class="list_simple">
+          <li>Tente adaptar as atividades de lazer para o ambiente de isolamento.</li>
+          <li>Se informe (apenas o necessário).</li>
+          <li>Tente manter a calma.</li>
+          <li>Pratique a paciência e a bondade.</li>
+          <li>Incentive outras pessoas a fazer o mesmo.</li>
+          <li>E lembre-se que estamos todos juntos!</li>
```
Comment (danzanzini) [inline app/views/ways_to_help/index.html.erb:152]: "E lembre-se: na união somos mais fortes!"

**Review by danzanzini — APPROVED** [review]
Great job both of you :top:

@hellofanny I liked The README structure. I just left some comments regarding the image path and the steps to initialize the project.

@renatamottam I've left some suggestions to make the content more gender-neutral.

File: app/views/ways_to_help/index.html.erb (hunk 4 above)
Comment (hellofanny) [reply app/views/ways_to_help/index.html.erb:152]: changed for "E lembre-se: a união faz a força!" 🙈

**Comment by hellofanny** [comment]
> Great job both of you 🔝
> 
> @hellofanny I liked The README structure. I just left some comments regarding the image path and the steps to initialize the project.
> 
> @renatamottam I've left some suggestions to make the content more gender-neutral.

Thank youuu! I made the adjustments ✨

**Pull request merged**